components/ILIAS/Forum/PRIVACY.md

```diff
@@ -0,0 +1,101 @@
+# Forum Privacy
+
+This documentation does not warrant completeness or correctness. Please report any
+missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](../../../docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+
+The forum is an ILIAS repository object in which users can hold discussions. A forum is organized in different topics (threads), to which contributions (posts) are written. As long as a forum is not deleted, posts can be viewed for years after they have been published.
+
+At least one person is responsible for moderation in all forums: They can censor and overwrite posts and close topics. Moderators can be added and removed by a person with the “Edit Settings” permission on the “Moderators” tab.
+
+## Configuration 
+
+Various privacy-related settings are made under “Administration > Repository and Objects > Forum”:
+
+**Enable Statistics in Forum** (default: off) sets for the installation whether a statistics function can be activated in the settings of individual forums.
```

Comment:
For clarification you can change:
"Enable Statistics in Forum (default: off) sets whether a statistics function can be activated in the settings of individual forums for the installation."